README.md

```diff
@@ -137,6 +137,16 @@ These comments can contain environment variables that will be translated when po
 
 This feature can be used to post for instance a URL to the deployed application or code coverage at a successful build and why the build failed like what tests that did not pass.
 
+## Pipeline support
+
+Support for Jenkins pipelines is currently experimental. To enable it, go to *Manage Jenkins*, then *Configure System* and check for *Enable Pipeline Support*.
+
+The configuration is the same as for other projects, but it is not currently possible to customize the comments posted to the Bitbucket Server after the job completion.
+
+Stash Pull Request Builder Plugin could serve as a stepping stone to a more advanced setup using Bitbucket Branch Source Plugin. Unlike the later, this plugin supports inline Groovy scripts for pipeline configuration, which may be handy while figuring out the optimal configuration for the build. Once that configuration is established, it can be added to the sources as `Jenkinsfile`.
+
```

Comment:
Maybe it would be useful to add two more points.
One is: when you create a new BS-based job it will try and build all (of the matching) branches in your repository. This might be troublesome when migrating a big project.

Another problem I would mention is  if you push to a new branch the BS plugin will probably start building it before you can make a PR out of it. This will result in two builds being run and two results being reported. 

Both of those are consequences of how the BS plugins are supposed to work, they are unlikely to change.

Reply:
OK, I'll add it.